Libraries/CustomComponents/ListView/ListView.js

```diff
@@ -229,6 +223,14 @@ var ListView = React.createClass({
      */
     removeClippedSubviews: React.PropTypes.bool,
     /**
+     * Makes the sections headers sticky.  The sticky behavior means that it
+     * will scroll with the content at the top of the section until it reaches
+     * the top of the screen, at which point it will stick to the top until it
+     * is pushed off the screen by the next section header.
+     * @platform ios
+     */
+    stickySectionHeaders: React.PropTypes.bool,
```

Comment:
@Kureev Previously the code would add in the indexes of all the sections headers whatever you set `stickyHeaderIndices` to. It did this by sending `this.props.stickyHeaderIndices.concat(sectionHeaderIndices)` to the `ScrollView` which the `ListView` uses. Because this happened within the `ListView`, there didn't seem to be a way to not have the headers sticky. The documentation also suggested this with the way it was worded. `stickyHeaderIndices` then essentially becomes the indexes of any _additional_ views which you want to be sticky. Without this flag, section headers are always passed to the underlying `ScrollView`.

This flag basically makes that single line of logic run conditionally only if you want the headers to be sticky.

I can't see how that block of code would disable the sticky headers. Where are you proposing using it?